websites/admin.py

```diff
@@ -42,12 +45,36 @@ class WebsiteContentAdmin(TimestampedModelAdmin):
     raw_id_fields = ("website", "parent")
 
 
+class WebsiteStarterForm(forms.ModelForm):
+    """Custom form for the WebsiteStarter model"""
+
+    class Meta:
+        model = WebsiteStarter
+        field_classes = {
+            "config": JsonOrYamlField,
+        }
+        fields = "__all__"
+
+    def __init__(self, *args, **kwargs):
+        kwargs_updated = {**kwargs, "error_class": WhitespaceErrorList}
+        super().__init__(*args, **kwargs_updated)
+
+    def clean_config(self):
+        """Ensures that the site config value obeys our schema"""
+        config = self.cleaned_data["config"]
+        try:
+            validate_parsed_site_config(config)
+        except ValueError as ex:
+            raise ValidationError(str(ex)) from ex
+        return config
+
+
 class WebsiteStarterAdmin(TimestampedModelAdmin):
     """WebsiteStarter Admin"""
 
     model = WebsiteStarter
 
-    formfield_overrides = {JSONField: {"widget": PrettyJSONWidget}}
+    form = WebsiteStarterForm
     include_created_on_in_list = True
     list_display = ("id", "name", "source", "commit")
```

Comment:
Can we add `slug` to this list, and to `search_fields` below? It would be useful to match the starter with messages from `override_site_configs` which describes the starter by the slug value